ShellScript/Bash.sublime-syntax

```diff
@@ -191,6 +192,22 @@ contexts:
 
 ###[ FUNCTION DEFINITIONS ]####################################################
 
+  def-anonymous:
+    - match: (\()\s*(\))
```

Comment:
I'm pretty sure, whitespaces aren't allowed between parentheses:

```zsh
% () { echo hi; }
hi
% (   ) { echo hi; }
zsh: parse error near `{'
```

```suggestion
    - match: (\()(\))
```

Reply:
Yes, but we might not want to break highlighting due to such details.

Reply:
You could make the `\s*` itself a capture group that is `illegal`

Reply:
We could, yes. But this context is a copy of the original implementation of arguments lists only. 

ST's auto match feature makes `( )` a very common pattern if we start editing groups or other things. To be honest, I don't like all the illegal highlighting too much. It may help to find really illegal patterns, but should not disturb a user too much while createing expressions. Incomplete expressions are illegal very often with regards to specs.

Reply:
If you think about it, most other editors and IDEs highlight illegal stuff with red underline or something like that — not as distracting as ST which just flashes red background all over the text.

Reply:
Underlining illegals is what I guess would happen with an LSP server running in the background.